### PR TITLE
Backend: OIDC client creds from env

### PR DESCRIPTION
**Pull request by yolossn**

### CODE REVIEW

File: backend/cmd/nebraska/nebraska.go [hunk 1]
```diff
@@ -29,6 +29,8 @@ const (
 	ghSessionCryptKeyEnvName   = "NEBRASKA_GITHUB_SESSION_CRYPT_KEY"
 	ghWebhookSecretEnvName     = "NEBRASKA_GITHUB_WEBHOOK_SECRET"
 	ghEnterpriseURLEnvName     = "NEBRASKA_GITHUB_ENTERPRISE_URL"
+	oidcClientIDEnvName        = "NEBRASKA_OIDC_CLIENT_ID"
```
Comment (joaquimrocha) [inline backend/cmd/nebraska/nebraska.go:34]: Nitpick for the commit body:
`**T**his patch allows the OIDC **c**lient credentials be
picked up from env**.**`

File: backend/cmd/nebraska/nebraska.go [hunk 2]
```diff
@@ -234,6 +245,20 @@ func obtainSessionOIDCCryptKey(potentialKey string) []byte {
 	return random.Data(32)
 }
 
+func obtainOIDCClientID(potentialID string) (string, error) {
```
Comment (joaquimrocha) [inline backend/cmd/nebraska/nebraska.go:250]: (comment for the future) Maybe we should look into using viper for making the config more flexible without this.

File: backend/cmd/nebraska/nebraska.go [hunk 3]
```diff
@@ -13,6 +13,8 @@ import (
 	"time"
 
 	"github.com/Depado/ginprom"
+	"github.com/gin-contrib/requestid"
```
Comment (joaquimrocha) [inline backend/cmd/nebraska/nebraska.go:16]: nipick on the commit body:
**T**his patch adds RequestID middleware
which sets a request id in response headers**.**

**Review by joaquimrocha — APPROVED** [review]

**Comment by joaquimrocha** [comment]
Code LGTM. Fix the nitpicks if you agree or just merge it.

File: backend/cmd/nebraska/nebraska.go (hunk 2 above)
Comment (yolossn) [reply backend/cmd/nebraska/nebraska.go:250]: Issue https://github.com/kinvolk/nebraska/issues/244 already exists, will take this up.

**Pull request merged**